Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -1081,8 +1081,12 @@ Device::profilerStop ()
 void hypreSynchronize ()
 {
 #ifdef AMREX_USE_GPU
+#if (HYPRE_RELEASE_NUMBER > 23200) || (HYPRE_RELEASE_NUMBER == 23200 && HYPRE_DEVELOP_NUMBER >= 4)
```

Comment:
```suggestion
#if (HYPRE_RELEASE_NUMBER > 23200)
```
Since this PR was submitted, there is a new hypre release, 2.33.0. So the logic here that deals with the hypre development version between 2.32 and 2.33 can be simplified now.